Loop/Managers/Alerts/AlertStore.swift

```diff
@@ -119,6 +125,29 @@ extension AlertStore {
 
 }
 
+// MARK: Alert Purging
+
+extension AlertStore {
+    var expireDate: Date {
```

Comment:
[nit] could be private?

Reply:
Pete requested that the "Simulated Core Data" code be separated from the rest of the code since it is primarily debug/test code. To accomplish that I pushed all related code for all stores into separate files named "<store>+SimulatedCoreData.swift" and added the code in an extension. Thus, this property must at least be internal.